### DISCUSSION TITLE
bg-gray-[###] not registering

### DISCUSSION THREAD

**Discussion by maxine** [tailwindlabs/tailwindcss#9905] [Help]
Hey! 

Here's a quick demo of the issue:
<img width="1191" alt="SCR-20221123-dxg" src="https://user-images.githubusercontent.com/23054741/203592828-9e898023-7655-492b-9503-bb439752e9e9.png">

You can see there is no background color getting applied when the background is set as `bg-gray-###`.

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/23054741/203593160-07d77123-5815-4660-97a8-54c8c678478c.png">

But it works when I change the color to `slate` instead of gray.

**Comment by wongjn** [comment]
Some things to check:

- Ensure you are not changing the class in the browser devtools – Tailwind generates CSS at *build time*, not at runtime.
- Ensure you are [not using string interpolation/concatenation to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).
- Ensure `gray.300` or `gray-300` exists as a color in the final, resolved Tailwind configuration.
- Ensure the file the code appears in is covered by the `content` globs in the Tailwind configuration.
- Ensure that Tailwind has run a compile since the last time you have updated the source code.

If all the above conditions pass, please consider providing a remotely-hosted git repository such that I could take a closer look at what might be wrong.
